Tests/UIKitTests/UIButtonExtensionsTests.swift

```diff
@@ -144,35 +144,93 @@ final class UIButtonExtensionsTests: XCTestCase {
         XCTAssertEqual(button.titleForSelected, title)
     }
 
-    func testCenterTextAndImage() {
+    func testSetImageLeftText() {
         let button = UIButton(frame: CGRect(x: 10, y: 10, width: 100, height: 100))
         let image = UIImage(color: .green, size: CGSize(width: 10, height: 10))
         button.setTitleForAllStates("Title")
         button.setImageForAllStates(image)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 10)
 
         XCTAssertNotNil(button.imageView)
         XCTAssertNotNil(button.titleLabel)
 
         let spacing: CGFloat = 20
 
         // Image on the left of text
-        button.centerTextAndImage(spacing: spacing)
-        var imageFrame = button.imageView!.frame
-        var titleFrame = button.titleLabel!.frame
+        button.setImageLeftText(spacing: spacing)
+        let imageFrame = button.imageView!.frame
+        let titleFrame = button.titleLabel!.frame
 
         XCTAssert(titleFrame.midX > imageFrame.midX)
         XCTAssertEqual(titleFrame.midY, imageFrame.midY, accuracy: 1.0)
         XCTAssertEqual(titleFrame.minX - spacing, imageFrame.maxX, accuracy: 1.0)
```

Comment:
Add a test for the left and right sides too. It should be something like:
```Swift
XCTAssertEqual(imageFrame.minX - spacing / 2, bounds.minX, accuracy: 1.0)
XCTAssertEqual(titleFrame.maxX + spacing / 2, bounds.maxX, accuracy: 1.0)
```

Reply:
I don't understand this test, `spacing / 2, bounds.minX` ?

Reply:
You want to check that the right side of the text reaches the bounds of the button, and that the left side of the image reaches the bounds. This is to check that they're positioned correctly and that the bounds are correct. This should help expose the issue of the content insets that I mentioned.

Reply:
I think there is no need to check that the right side of the text reaches the bounds of the button and ...
```
XCTAssertEqual(imageFrame.minX - spacing / 2, bounds.minX, accuracy: 1.0)
```
And the checks should not be equal.